### ISSUE TITLE
Use of icc() for multivariate brms models?

### ISSUE THREAD

**Issue by jrosen48** [sjPlot/sjstats#76]
thank you *again for creating such a useful set of tools. 

Can you say how `sjstats::icc()` works for mutlivariate `brmsfit` models (e.g., [here](https://cran.r-project.org/web/packages/brms/vignettes/brms_multivariate.html))?

**Comment by strengejacke** [comment]
What do you mean by "how"? How it works without getting an error, or how it works regarding the underlying statistics?

Example:

``` r
m <- insight::download_model("brms_mv_2")
sjstats::icc(m, ppd = TRUE)
#> 
#> # Random Effect Variances and ICC
#> 
#> ## Variance Ratio (comparable to ICC)
#> Ratio: -0.04  HDI 89%: [-0.45 0.39]
#> 
#> ## Variances of Posterior Predicted Distribution
#> Conditioned on fixed effects: 7.27  HDI 89%: [-7.18 37.76]
#> Conditioned on rand. effects: 7.81  HDI 89%: [-6.66 39.54]
#> 
#> ## Difference in Variances
#> Difference: 0.54  HDI 89%: [-4.39 8.76]
```

<sup>Created on 2019-04-02 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>


From the help:

> ICC for Bayesian models 
> 
> If ppd = TRUE, icc() calculates a variance decomposition based on the posterior predictive distribution. In this case, first, the draws from the posterior predictive distribution not conditioned on group-level terms (posterior_predict(..., re.form = NA)) are calculated as well as draws from this distribution conditioned on all random effects (by default, unless specified else in re.form) are taken. Then, second, the variances for each of these draws are calculated. The "ICC" is then the ratio between these two variances. This is the recommended way to analyse random-effect-variances for non-Gaussian models. It is then possible to compare variances accross models, also by specifying different group-level terms via the re.form-argument. 
> 
> Sometimes, when the variance of the posterior predictive distribution is very large, the variance ratio in the output makes no sense, e.g. because it is negative. In such cases, it might help to use a more robust measure to calculate the central tendency of the variances. For example, use typical = "median".

**Comment by strengejacke** [comment]
Note that the Bayes-functions are re-implemented in the [bayestestR-package](https://github.com/easystats/bayestestR), which is short before release.

The functions related to "model performance" (like R2, ICC etc.) will be re-implemented in the [performance-package](https://github.com/easystats/performance), which is planned for release once bayestestR is on GitHub.

However, for brms-models, we decided to only report the ICC based on variance decomposition in the _performance_ package (i.e. when you set `ppd = TRUE` in `sjstats::icc()`, because the other ICC calculations were too unstable for most of the models you can fit with brms.
